java/src/jmri/jmrit/Bundle_cs.properties

```diff
@@ -29,9 +29,9 @@ MenuItemSignalMastTable = N\u00e1v\u011bstidla
 MenuItemSignalGroupTable = N\u00e1v\u011bstn\u00ed skupiny
 MenuItemSignalMastLogicTable = Logika n\u00e1v\u011bstidel
 MenuItemRailComTable=RailCom ID
-MenuItemLogixNGTable = LogixNG
-MenuItemLogixNGModuleTable = LogixNG Modul
-MenuItemLogixNGTableTable = LogixNG Tabulka
+MenuItemLogixNGTable = LogixyNG
```

Comment:
Is this change intentional? LogixNG -> LogixyNG

Reply:
Changed was default properties file

https://github.com/JMRI/JMRI/blob/b3bb8f8483d30096fcdf2b587dfcf9c34f7e61e3/java/src/jmri/jmrit/Bundle.properties#L32

Logix / Logixs ~ singular / plural
Logix / Logixy ~ singular / plural, for better pronunciation/meaning I moved _y_ between Logix and NG 

I translate in the same way 
ConditionalNG / ConditionalNGs
PodmínkaNG / PodmínkyNG

Reply:
Thanks.